### PR TITLE
For intalling amabariserver

### PR DESCRIPTION
**Pull request by StayerX**
Creating a new pull request for the amabri setup.
@CloudwickShared/cloudwick do not pull until everyone has agreed (all 5 members).

### CODE REVIEW

File: ambari-server.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+cd /etc/yum.repos.d/
+apt-get http://public-repo-1.hortonworks.com/ambari/centos6/2.x/updates/2.2.1.0/ambari.repo
```
Comment (StayerX) [inline ambari-server.sh]: Should be `wget` not `apt-get`.

File: ambari-server.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+cd /etc/yum.repos.d/
```
Comment (StayerX) [inline ambari-server.sh:2]: Should have `#!/bin/bash above` this line

Comment (StayerX) [inline ambari-server.sh:2]: Should have `rm /etc/yum.repos.d/ambari.repo` bellow this line.

**Comment by StayerX** [comment]
@taffazzel we will see this together at work.

**Comment by StayerX** [comment]
@bhavintandel @despoinak1 @duhit How does it look to you? To me it seems good.

**Comment by bhavintandel** [comment]
The last line, @taffazzel has written twice the setup command

**Comment by despoinak1** [comment]
It seems good. please add echo messages so that the user knows what each commands does.

**Pull request merged**